src/ingest_validation_tools/table-schemas/assays/czems-v1.yaml

```diff
@@ -0,0 +1,133 @@
+doc_url: https://portal.hubmapconsortium.org/docs/assays/maldi-ims
+fields:
+# include: ../includes/vA/all_assays.yaml
+- name: assay_category
+  constraints:
+    enum:
+      - mass_spectrometry
+- name: assay_type
+  constraints:
+    enum:
+      - CZE-MS
+- name: analyte_class
+  constraints:
+    enum:
+      - protein # NOTE: "proteins" specified, but singular is used elsewhere.
+      - metabolites
+      - lipids
+      - peptides
+      - phosphopeptides
+      - glycans
+
+# include: ../includes/v0/is_targeted.yaml
+
+# Level 2:
+
+# include: ../includes/v0/acquisition_instrument.yaml
+- name: ms_source
+  description: The technique used for sampling and ionization of the sample.
+  constraints:
+    enum:
+      - nanoflex
+      - ESI
+      - ESI
+- name: polarity
+  description: The polarity of the mass analysis (positive or negative ion modes)
+  constraints:
+    enum:
+      - negative ion mode
+      - positive ion mode
+- name: mz_range_low_value
+  description: The low value of the scanned mass range for MS1. (unitless)
+- name: mz_range_high_value
+  description: The high value of the scanned mass range for MS1. (unitless)
+
+- name: mass_resolving_power
+  description: The MS1 resolving power defined as m/∆m where ∆m is the FWHM for a given peak with a specified m/z (m). (unitless)
+  # TODO: Not the same definitions as given in IMS. Is that correct?
```

Comment:
```suggestion
  description: This is the MS1 resolving power. This is a unitless value often calculated as m/∆m where ∆m is the FWHM for a given peak with a certain m/z (m).
```

Reply:
Copied the description text from IMS and pasted here to make the descriptions identical.